spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+feature 'User sign up' do
+
+  let(:user){build :user}
+
+  def sign_up(user)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for sign_up is too high. [17/15]